assets/src/helpers/detectSwipe.ts

```diff
@@ -0,0 +1,79 @@
+import { SwipeDirection } from "../skate.d"
+
+const MIN_X = 30 // Min x swipe for horizontal swipe
+const MAX_X = 30 // Max x difference for vertical swipe
+const MIN_Y = 50 // Min y swipe for vertical swipe
+const MAX_Y = 60 // Max y difference for horizontal swipe
+
+type Callback = (swipeDirection: SwipeDirection) => void
+
+interface Position {
+  x: number
+  y: number
+}
+
+const detectSwipe = (elementId: string, callback: Callback) => {
+  let startingPosition: Position
+  let endingPosition: Position
+
+  const handleTouchStart = (event: TouchEvent) => {
+    event.preventDefault()
+    startingPosition = firstTouchPosition(event)
+  }
```

Comment:
Does `handleTouchStart` also need to set `endingPosition` to avoid a bug? If you `touchstart` and `touchend` without any `touchmove`s inbetween, could you get an `endingPosition` that's unset or has a value from the previous swipe?

Alternatively, can you get the endingPosition out of the event in `handleTouchEnd` and not need a `touchmove` handler or stateful `let endingPosition` at all?

Reply:
We can play with endingPosition more too if we want, but it doesn't really matter since if they haven't moved it won't be a swipe.